operators/opendatahub-operator/2.19.0/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1227 @@
+                - name: OPERATOR_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: DEFAULT_MANIFESTS_PATH
+                  value: /opt/manifests
+                - name: ODH_PLATFORM_TYPE
+                  value: OpenDataHub
+                image: quay.io/opendatahub/opendatahub-operator:v2.19.0
+                imagePullPolicy: Always
+                livenessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 8081
+                  initialDelaySeconds: 15
+                  periodSeconds: 20
+                name: manager
+                ports:
+                - containerPort: 9443
+                  name: webhook-server
+                  protocol: TCP
+                - containerPort: 8080
+                  name: http
+                  protocol: TCP
+                readinessProbe:
+                  httpGet:
+                    path: /readyz
+                    port: 8081
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
+                resources:
+                  limits:
+                    cpu: 500m
+                    memory: 4Gi
+                  requests:
+                    cpu: 100m
+                    memory: 780Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+                  capabilities:
+                    drop:
+                    - ALL
+                volumeMounts:
+                - mountPath: /tmp/k8s-webhook-server/serving-certs
+                  name: cert
+                  readOnly: true
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: opendatahub-operator-controller-manager
+              terminationGracePeriodSeconds: 10
+              volumes:
+              - name: cert
+                secret:
+                  defaultMode: 420
+                  secretName: opendatahub-operator-controller-webhook-cert
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - odh
+  - notebooks
+  - serving
+  - training
+  - pipelines
+  - modelmesh
+  - workbenches
+  - dashboard
+  - kserve
+  - distributed workloads
+  - trustyai
+  - modelregistry
+  links:
+  - name: Open Data Hub
+    url: https://opendatahub.io
+  - name: Open Data Hub Community
+    url: https://github.com/opendatahub-io/opendatahub-community
+  maintainers:
+  - email: opendatahub@redhat.com
+    name: Open Data Hub Maintainers
+  maturity: stable
+  minKubeVersion: 1.25.0
+  provider:
+    name: ODH
+  selector:
+    matchLabels:
+      component: opendatahub-operator
+  skips:
+    - opendatahub-operator.v2.9.0
+  replaces: opendatahub-operator.v2.18.2
```

Comment:
why we need this if we already have olm.skipRange?

Reply:
hmm maybe this is not required at all but I am not sure. I took inspiration from the older manifests files like: https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/operators/opendatahub-operator/2.17.0/manifests/opendatahub-operator.clusterserviceversion.yaml#L1801

Reply:
ok, lets keep it here, if it is only not necessary for RH commuinty and operatorhub, others might need it to work with CSV.

Reply:
ok sure 👍